micro-rdk-installer/src/nvs/request.rs

```diff
@@ -15,6 +19,22 @@ This module contains the logic for acquiring the security credentials for a robo
 the Viam App and preparing for flash storage
 */
 
+pub async fn download_latest_release(tmp_dir: &TempDir) -> Result<PathBuf, Error> {
+    let release_url = "https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-esp32-server.bin";
```

Comment:
this should be an option too, by default it should point to release but users should be able to point to a specific version too. I don't know how github deals with releases but passing a version flag with version=latest or x.x.x would be perfect